taskmanager/spec/Application/Task/StatusSpec.php

```diff
@@ -34,4 +34,22 @@ function it_should_has_todo_static_contstructor()
     {
         $this->shouldReturnAnInstanceOf(Status::class);
     }
+
+    function it_should_has_in_progress_static_contstructor()
```

Comment:
@mateuszmaslanka as this is for code modeling and not testing do we need all of them? or just one example status?

Reply:
fixed - replaced with one function